index.html

```diff
@@ -172,23 +173,49 @@ <h4 class="title-4 m-0">
         </ul>
       </section>
     </main>
-    <footer>
-      <ul>
-        <li>Home</li>
-        <li>About</li>
-        <li>logo</li>
-        <li>Service</li>
-        <li>Blog</li>
-      </ul>
-      <hr />
-      <ul>
-        <li></li>
-        <li></li>
-        <li></li>
-        <li></li>
-        <li></li>
-      </ul>
-      <p></p>
+    <footer class="footer">
+      <section class="container">
+        <nav class="footer-nav">
+          <ul class="footer-nav-list">
+            <li>
+              <a class="footer-nav-list-link" href="">Home</a>
+            </li>
+            <li>
+              <a class="footer-nav-list-link" href="">About</a>
+            </li>
+            <li>
+              <a class="footer-nav-list-link" href="">
+                <img src="images/logo-footer.svg" alt="logo" />
+              </a>
+            </li>
+            <li>
+              <a class="footer-nav-list-link" href="">Service</a>
+            </li>
+            <li>
+              <a class="footer-nav-list-link" href="">Blog</a>
+            </li>
+          </ul>
+        </nav>
+        <hr class="horizontal-line" />
+        <ul class="social-network-list">
+          <li>
+            <a href=""><span class="icon-facebook icon-footer"></span></a>
+          </li>
+          <li>
+            <a href=""><span class="icon-twitter icon-footer"></span></a>
+          </li>
+          <li>
+            <a href=""><span class="icon-instagram icon-footer"></span></a>
+          </li>
+          <li>
+            <a href=""><span class="icon-youtube icon-footer"></span></a>
+          </li>
+          <li>
+            <a href=""><span class="icon-pinterest icon-footer"></span></a>
+          </li>
+        </ul>
+        <p class="copyright">© 2010 — 2020 Privacy — Terms</p>
```

Comment:
Use special character for copyright https://www.teachucomp.com/special-characters-in-html-tutorial/